src/test/settings_test.ts

```diff
@@ -47,15 +47,36 @@ suite('Settings', () => {
     });
     const settingsChange = await settings.changeStream.next;
     assert.deepEqual(settingsChange, {
-      newer: {fixOnSave: true, analyzeWholePackage: true},
-      older: {fixOnSave: false, analyzeWholePackage: false}
+      newer: {
+        fixOnSave: true,
+        analyzeWholePackage: true,
+        referencesCodeLens: false,
+        logToClient: false,
+        logToFile: undefined
+      },
+      older: {
+        fixOnSave: false,
+        analyzeWholePackage: false,
+        referencesCodeLens: false,
+        logToClient: false,
+        logToFile: undefined
```

Comment:
can you add a test for `logToFile !== undefined`?

Reply:
+1, done